kaggle_environments/envs/lux_ai_2022/test_agents/python/lux/config.py

```diff
@@ -0,0 +1,142 @@
+    MIN_FACTORIES: int = 2
+    MAX_FACTORIES: int = 5
+    CYCLE_LENGTH: int = 50
+    DAY_LENGTH: int = 30
+    UNIT_ACTION_QUEUE_SIZE: int = 10 # when set to 1, then no action queue is used
+    UNIT_ACTION_QUEUE_POWER_COST: Dict[str, int] = dataclasses.field(default_factory=lambda: dict(
+        LIGHT=1,
+        HEAVY=10
+    ))
+    # DEPRECATED
+    # UNITS_CONTROLLED: int = 20 # when set to -1, all units can be controlled at once
+
+
+    MAX_RUBBLE: int = 100
+    FACTORY_RUBBLE_AFTER_DESTRUCTION: int = 50
+    INIT_WATER_METAL_PER_FACTORY: int = 100 # amount of water and metal units given to each factory
+    INIT_POWER_PER_FACTORY: int = 100
+
+    #### LICHEN ####
+    MIN_LICHEN_TO_SPREAD: int = 1
+    LICHEN_LOST_WITHOUT_WATER: int = 1
+    LICHEN_GAINED_WITH_WATER: int = 1
+    LICHEN_WATERING_COST_FACTOR: int = 10
+
+    #### Bidding System ####
+    BIDDING_SYSTEM: bool = True
+
+    #### Factores ####
+    FACTORY_PROCESSING_RATE_WATER: int = 100
+    ICE_WATER_RATIO: int = 5
+    FACTORY_PROCESSING_RATE_METAL: int = 50
+    ORE_METAL_RATIO: int = 5
+    FACTORY_CHARGE: int = 50
+    FACTORY_WATER_CONSUMPTION: int = 1
+
+
+    #### Units ####
+    ROBOTS: Dict[str, UnitConfig] = dataclasses.field(
+        default_factory=lambda: dict(
+        LIGHT=UnitConfig(
+            METAL_COST=10, POWER_COST=50, INIT_POWER=50, CARGO_SPACE=100, BATTERY_CAPACITY=50, CHARGE=1, MOVE_COST=1, RUBBLE_MOVEMENT_COST=1,
+            DIG_COST=5,
+            SELF_DESTRUCT_COST=5,
+            DIG_RUBBLE_REMOVED=1,
+            DIG_RESOURCE_GAIN=2,
+            DIG_LICHEN_REMOVED=10,
+            RUBBLE_AFTER_DESTRUCTION=1,
+        ),
+
+        HEAVY=UnitConfig(
+            METAL_COST=100, POWER_COST=500, INIT_POWER=500, CARGO_SPACE=1000, BATTERY_CAPACITY=1500, CHARGE=10, MOVE_COST=20, RUBBLE_MOVEMENT_COST=1,
+            DIG_COST=100,
+            SELF_DESTRUCT_COST=100,
+            DIG_RUBBLE_REMOVED=10,
+            DIG_RESOURCE_GAIN=20,
+            DIG_LICHEN_REMOVED=100,
+            RUBBLE_AFTER_DESTRUCTION=10,
+        ),
+        )
+    )
+
+    #### Map Generation ####
+    # TODO
+
+    #### Weather ####
+    WEATHER_ID_TO_NAME: list = dataclasses.field(default_factory=lambda : ["NONE","MARS_QUAKE","COLD_SNAP","DUST_STORM","SOLAR_FLARE"])
+    NUM_WEATHER_EVENTS_RANGE: List[int] = dataclasses.field(default_factory=lambda:[3,5])
+    WEATHER: dict = dataclasses.field(default_factory=lambda:dict(
+        MARS_QUAKE=dict(
+            # amount of rubble generated under each robot per turn
+            RUBBLE=dict(LIGHT=1, HEAVY=10),
+            TIME_RANGE=[1, 5]
+        ),
+        COLD_SNAP=dict(
+            # power multiplier required per robot action. 2 -> requires 2x as much power to execute the same action
+            POWER_CONSUMPTION=2,
+            TIME_RANGE=[10, 30]
+        ),
+        DUST_STORM=dict(
+            # power multiplier required per robot action. .5 -> requires .5x as much power to execute the same action
```

Comment:
I thought this was gain .5x as much power per turn (dust makes it harder to gain power?)

Reply:
oops good catch